package.json

```diff
@@ -30,13 +31,17 @@
     "@ionic/angular": "^5.1.0",
     "@ionic/pwa-elements": "^1.5.2",
     "@logisticinfotech/ionic4-datepicker": "^1.4.3",
+    "@ng-idle/core": "^8.0.0-beta.4",
+    "@ng-idle/keepalive": "^8.0.0-beta.4",
```

Comment:
Can we check if there is stable version for idle? If yes, please update dependencies to stable version.

Reply:
https://www.npmjs.com/package/@ng-idle/core
https://www.npmjs.com/package/@ng-idle/keepalive

All versions are stable. They use that kind of name for all version. Check those links.